### ISSUE TITLE
Packages incompatibility, - impossible to upgrade "dart pub get / - null safety  "

### ISSUE THREAD

**Issue by momojdy** [GanZhiXiong/GZXTaoBaoAppFlutter#43]
Hello are you still maintaining this project ?

**Comment by ay2853** [comment]
邮件已经收到，我会尽快查收！

**Comment by GanZhiXiong** [comment]
I won't maintain it anymore because I don't have the time now and it won't bring me any profit.
If there is a business cooperation, I can consider it.

**Comment by momojdy** [comment]
Yes a business collaboration is on the table if you open to that . How can I contact you ?

**Comment by GanZhiXiong** [comment]
@momojdy 

Yes, you can send me your requirements by email 1551935335@qq.com

**Comment by momojdy** [comment]
@GanZhiXiong please do let me know if you have received my email , I also attached my wechat for easier communication.
